### PR TITLE
Fixed Stackable Flag For Custom Items

### PR DESCRIPTION
**Pull request by Wolftein**
Don't know if this flag was never implemented, but never works, so i just fixed it.

Signed-off-by: Wolftein agustin.l.alvarez@hotmail.com

### CODE REVIEW

File: src/main/java/org/getspout/spout/item/mcitem/CustomItemFlint.java [hunk 1]
```diff
@@ -60,6 +61,8 @@ public ItemStack b(ItemStack itemstack, World world, EntityHuman entityhuman) {
 		if (item instanceof Food) {
 			--itemstack.count;
 			entityhuman.getFoodData().eat(((Food) item).getHungerRestored(), 0.6F);
+		} else if(item instanceof Weapon) {
```
Comment (ZNickq) [inline src/main/java/org/getspout/spout/item/mcitem/CustomItemFlint.java:64]: Is that part of the weapon PR? o.O

Comment (Wolftein) [inline src/main/java/org/getspout/spout/item/mcitem/CustomItemFlint.java:64]: Don't know why this happend, this is an entire different commit :S

Comment (stephenwilliams) [inline src/main/java/org/getspout/spout/item/mcitem/CustomItemFlint.java:64]: When you submit a PR, any commits pushed to the branch that the PR was submitted from will be added to the PR

Comment (Wolftein) [inline src/main/java/org/getspout/spout/item/mcitem/CustomItemFlint.java:64]: Good to know :) thanks Alta189

**Comment by Wolftein** [comment]
Cancel until Legacy team pulls a recommended build!

**Pull request closed without merging**